docs/public-documentation/Output.md

```diff
@@ -77,11 +282,288 @@ For connection to a secure Kafka Broker -
 
 # JavaScript/TypeScript/Python/Powershell
 
+
+### JavaScript
+
 The following example shows a Kafka output binding in a _function.json_ file and a [JavaScript function](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-node) that uses the binding. The function reads in the message from an HTTP trigger and outputs it to the Kafka topic.
 
-Here&#39;s the binding data in the _function.json_ file:
+Here is the binding data in the _function.json_ file:
+
+```json
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": [
+        "get"
+      ]
+    },
+    {
+      "type": "kafka",
+      "name": "outputKafkaMessage",
+      "brokerList": "BrokerList",
+      "topic": "topic",
+      "username": "ConfluentCloudUsername",
+      "password": "ConfluentCloudPassword",
+      "protocol": "SASLSSL",
+      "authenticationMode": "PLAIN",
+      "direction": "out"
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "res"
+    }
+  ]
+}
+```
+
+Here is JavaScript code:
+
+```js
+module.exports = async function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    const message = (req.query.message);
+    context.bindings.outputKafkaMessage = message;
+    context.res = {
+        body: 'Ok'
+    };
+}
+```
+
+Here's JavaScript code that sends multiple messages:
```

Comment:
Please add the changes required in function.json for batch events

Reply:
No changes are requires in function.json for batch events in output. function.json is the same for batch events and single events.